src/python/nimbusml/internal/core/linear_model/averagedperceptronbinaryclassifier.py

```diff
@@ -107,27 +107,22 @@ class AveragedPerceptronBinaryClassifier(
 
     :param decrease_learning_rate: Decrease learning rate.
 
-    :param l2_regularizer_weight: L2 Regularization Weight.
+    :param l2_regularization: L2 Regularization Weight.
 
-    :param num_iterations: Number of iterations.
+    :param number_of_iterations: Number of iterations.
 
-    :param init_wts_diameter: Sets the initial weights diameter that specifies
-        the range from which values are drawn for the initial weights. These
-        weights are initialized randomly from within this range. For example,
-        if the diameter is specified to be ``d``, then the weights are
-        uniformly distributed between ``-d/2`` and ``d/2``. The default value
-        is ``0``, which specifies that all the  weights are set to zero.
```

Comment:
Looks like description for initial_weights_diameter is regressing to something useless. Could you please check why this is happening?

Reply:
Thanks good catch. Due to param renaming I had to update manifest_diff.json with new param names so the description is picked up

---
In reply to: [285733648](https://github.com/microsoft/NimbusML/pull/100#discussion_r285733648) [](ancestors = 285733648)